mreg/apps.py

```diff
@@ -1,8 +1,22 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.db.models.signals import post_migrate
 
 
 class MregAppConfig(AppConfig):
     name = 'mreg'
 
     def ready(self):
         import mreg.signals # noqa
+        from mreg.models.network_policy import NetworkPolicyAttribute
+
+        def create_protected_attributes(sender, **kwargs):
+            protected_attrs = getattr(settings, 'MREG_PROTECTED_POLICY_ATTRIBUTES', [])
+            for attr in protected_attrs:
+                NetworkPolicyAttribute.objects.get_or_create(
+                    name=attr,
+                    defaults={'description': 'Automatically created protected attribute.'}
+                )
```

Comment:
In the absence of any centralized config validation, we should check if we have a valid configuration here before creating the protected attributes. This prevents misconfiguration of the application such as:


```py
MREG_PROTECTED_POLICY_ATTRIBUTES = "isolated
```

Creating the attributes `i`, `s`, `o`, `l`, `a`, `t`, `e`, `d`.

```suggestion
        def create_protected_attributes(sender, **kwargs):
            protected_attrs = getattr(settings, 'MREG_PROTECTED_POLICY_ATTRIBUTES', [])

            if not isinstance(protected_attrs, list):
                raise ValueError('Config option MREG_PROTECTED_POLICY_ATTRIBUTES must be a list.')
            
            for attr in protected_attrs:
                NetworkPolicyAttribute.objects.get_or_create(
                    name=str(attr),
                    defaults={'description': 'Automatically created protected attribute.'}
                )
```

Reply:
Fixed universally.